crates/net/downloaders/src/headers/linear.rs

```diff
@@ -21,6 +21,7 @@ use std::{
 };
 
 /// Download headers in batches
+// ANCHOR: struct-LinearDownloader
```

Comment:
I don't think it's worth pursuing at the moment, since we'll be moving this content to `docs` anyway and will remove all the anchors and replace with code blocks